README.md

```diff
@@ -75,37 +75,45 @@ Usage: ruby ticketmatch.rb [options]
     -h, --help                       this message
 ```
 
-# Batch processing
+# Batch processing for puppet-agent and its components
 
-If you need to check all the components that comprise `puppet-agent` for a release, use
-`pa_matchbatch.sh`. It requires one argument: the revision (SHA or reference) of `puppet-agent` you
-care about. It will process `puppet-agent` and all its components that are tagged with a SHA.
 
-Component directories will be cloned from the path provided in their `component.json` file and the
-appropriate branch set. If a given component already exists, a Git fetch will be performed to get
-things up to date (see environment variables, below, for additional options).
+The `pa_matchbatch.sh` script clones a revision of puppet-agent and all of its components, then runs
+ticketmatch on everything; This is intended for use during puppet-agent releases.
 
-## How to run pa_matchbatch (one way)
+### How to run pa_matchbatch
 
-set your working directory to someplace (cannot be `puppet-agent`; can be its parent or empty)
+*`pa_matchbatch.sh` requires one argument,* A puppet-agent git reference (a branch name, a SHA, or
+a tag). For example, to run ticketmatch on puppet-agent's master branch and all of its components:
 
-```cd <a_directory>```
-
-invoke pa_matchbatch
+```sh
+./pa_matchbatch.sh <branch-name>
+```
 
-```path/pa_matchbatch.sh <the_revision>```
+*`pa_matchbatch.sh` clones puppet-agent and its components to the current working directory by default.*
+You can override the target directory by setting the `$WORKSPACE` environment variable to some other
+directory. You can run `pa_matchbatch.sh` any number of times on the same workspace; The repos will
+be fetched each time to check for new updates.
 
-## How to run pa_matchbatch (another way)
+*`pa_matchbatch.sh` assumes that only the Z versions of puppet-agent and its components will change
+during a release.* You will need to override this assumption for:
 
-point the WORKSPACE environment variable to a directory (ditto)
+- Any update to an X or Y version number, or
+- Any circumstance where the reported version number is incorrect for some reason (a bad mergeup, for example)
 
-```export WORKSPACE="fully_qualified_path_to_a_dir"```
+To override the assumed version numbers for some or all components, create a text file with one
+`<fixversion-prefix-for-jira-project>:<version>` entry on each line. For example, this would
+override the assumed versions of puppet-agent, facter, and pxp-agent:
 
-invoke pa_matchbatch
+```
+PUP:6.6.6
+FACT:3.13.0
+pxp-agent:1.11.0    
```

Comment:
This format seems a little weird (using the jira fixVersion prefixes instead of the component names), but it seems like changing the behavior should be a separate PR.